tests/h/search/query_test.py

```diff
@@ -496,6 +496,117 @@ def storage(self, patch):
         return patch('h.search.query.storage')
 
 
+class TestUriCombinedWildcardFilter():
+
+    @pytest.mark.parametrize('params,expected,separate_keys', [
```

Comment:
It wasn't immediately clear what `expected` is here. IDs? Indexes? Something else?